doc/persistent_scenes_and_states/persistent_scenes_and_states.rst

```diff
@@ -0,0 +1,59 @@
+Persistent Planning Scenes and Robot States
+===========================================
+
+The "MotionPlanning" plugin of RViz offers the possibility to save
+complete planning scenes and robot states persistently.
+Currently, two storage plugins (based on
+`warehouse_ros <https://github.com/ros-planning/warehouse_ros>`_) are available:
+
+* `warehouse_ros_mongo <https://github.com/ros-planning/warehouse_ros_mongo>`_, which uses MongoDB as backend (default)
```

Comment:
I agree, but then we have to change all the demo files and the templates for the MSA. This would be a breaking change, maybe postpone it to the next distro (H Turtle)?

Reply:
There is no official MSA for MoveIt2 yet, so we still have all the flexibility there as far as I know. :)
But yes, in MoveIt the config templates mention mongodb in multiple places, so de fact that makes it the default.
However, I would still prefer to remove the `(default)` part to not bias potential users into thinking mongodb might be better.